python/mujinplanningclient/binpickingplanningclient.py

```diff
@@ -79,29 +79,28 @@ def PickAndPlace(
         """Picks up an object with the targetnamepattern and places it down at one of the goals. First computes the entire plan from robot moving to a grasp and then moving to its destination, then runs it on the real robot. Task finishes once the real robot is at the destination.
 
         Args:
-            goaltype (str): (deprecated) type of the goal, e.g. translationdirection5d or transform6d
-            goals (str): (deprecated) flat list of goals, e.g. two 5d ik goals: [380,450,50,0,0,1, 380,450,50,0,0,-1]
-
-            targetnamepattern (str): regular expression describing the name of the object. No default will be provided, caller must set this. See https://docs.python.org/2/library/re.html
-            approachoffset (float, optional): Distance in millimeters to move straight to the grasp point, e.g. 30 mm
-            departoffsetdir (list[float], optional): The direction and distance in mm to move the part in global frame (usually along negative gravity) after it is grasped, e.g. [0,0,50]
-            destdepartoffsetdir (list[float], optional): The direction and distance in mm to move away from the object after it is placed, e.g. [0,0,30]. Depending on leaveoffsetintool parameter, this can in the global coordinate system or tool coordinate system.
-            deletetarget (int, optional): whether to delete target after pick and place is done
-            debuglevel (str): sets debug level of the task
-            movetodestination (str): planning parameter
-            freeinc (str): planning parameter
-            worksteplength (str): planning parameter
-            densowavearmgroup (str): planning parameter
-            regionname (str, optional): Name of the region of the objects
+            goaltype (str, optional): **deprecated** Type of the goal, e.g. translationdirection5d or transform6d
+            goals (str, optional): **deprecated** Flat list of goals, e.g. two 5d ik goals: [380,450,50,0,0,1, 380,450,50,0,0,-1]
```

Comment:
These are not optional in the function signature. IIRC we have a convention like this in robotbridgeclient for deprecated args:

```
_deprecated = None
[...]
   def PickAndPlace(
         self,
         goaltype=_deprecated,
         goal=_deprecated,
...
```

Reply:
Yes, but this PR was intended to only change the docstrings.  I see though that you've added the `_deprecated`. Lets keep it, I dont mind. Thanks.